### PR TITLE
7.6 list slicing, shanxiu

### PR DESCRIPTION
**Pull request by heshanxiu**
finish list slicing implementation, please review, thank you so much Harsha!

### CODE REVIEW

File: tests/test_list_slicing.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    ([1, 2, 3, 4, 5], 2, -1)
+    ([1, 2, 3, 4, 5], 3, 3)
+]
+
+max_score = len(test_params)
+
+function_name = "slice_list" # TODO
+
+# TODO:  name the class according to the function name being tested
+class GetDictionaryValue_Test(BaseClass):
```
Comment (harsha-miryala) [inline tests/test_list_slicing.py]: Can you change the name of the class here

Comment (heshanxiu) [reply tests/test_list_slicing.py]: hi, thank you for catching that! just updated it

Comment (heshanxiu) [reply tests/test_list_slicing.py]: If no other problems, should be able to merge

Comment (harshaucsb) [reply tests/test_list_slicing.py]: Thanks Shanxiu .. I will merge this now

**Review by harshaucsb — APPROVED** [review]

**Pull request merged**